_posts/2020-04-17-snowplow-ios-tracker-1.3.0-released/index.md

```diff
@@ -0,0 +1,309 @@
+|        `documentVersion` | Version of the document        | No            | String   |
+| `documentDescription` | Description of the document | No            | String   |
+
+The required basisForProcessing accepts only the following literals: `consent`, `contract`, `legal_obligation`, `vital_interests`, `public_task`, `legitimate_interests` - in accordance with the [five legal bases for processing](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/lawful-basis-for-processing/).
+
+The GDPR context is enabled by calling the `setGdprContextWithBasis:documentId:documentVersion:documentDescription:` method of the tracker builder or by calling the `enableGdprContextWithBasis:documentId:documentVersion:documentDescription:` method once the tracker has been initialised.
+
+It is called as follows:
+
+{% highlight objc %}
+SPTracker *tracker = [SPTracker build:^(id<SPTrackerBuilder> builder) {
+    [builder setEmitter:emitter];
+    ...
+    [builder setGdprContextWithBasis:SPGdprProcessingBasisConsent
+                          documentId:@"someId"
+                     documentVersion:@"0.1.0"
+                 documentDescription:@"a demo document description"];
+}];
+{% endhighlight %}
+
+<h2 id="spm">3. Swift Package Manager support</h2>
+
+The Swift Package Manager is the official Apple dependency manager designed for Swift libraries. Since Xcode 11, SPM is compatible with the iOS build system.
+Thanks to the important contribution from [@vauxall][contributor], we have introduced the support of Swift Package Manager for our iOS tracker ([#474][474]). At the moment it supports only the iOS platform due to some limitations that we will resolve in future releases, but it is a great option for adding our mobile tracker and a great contribution from our active community.
+
+To install Snowplow Tracker with SPM:
+1. In Xcode, select File > Swift Packages > Add Package Dependency
+2. Add the repository of: https://github.com/snowplow/snowplow-objc-tracker
+3. Add the modules to your project's targets in the General tab > _Frameworks, Libraries & Embedded Content_ section
+ 
+<h2 id="deprecated">4. Deprecated methods</h2>
+
+In this release we've marked a few methods as deprecated:
+
+- SPEvent - `eventId`: This shouldn't be used as it can cause ambiguity about event duplication. This property will be set only by the tracker. If your code sets its own eventId we suggest to record your custom event ID in a custom context associated to all the events.
+
+- SPEvent - `timestamp`: In a future major version we will remove the ability to override the timestamp of the event. That will be set only by the tracker. A custom timestamp can be eventually added using a different method or parameter in a future release.
+
+- SPEvent - `addDefaultParamsToPayload:`: This shouldn't be used as it's an internal method used by the tracker and it can be subject of future changes. 
```

Comment:
```suggestion
- SPEvent - `addDefaultParamsToPayload:`: This shouldn't be used as it's an internal method used by the tracker and it can be subject to future changes. 
```